### ISSUE TITLE
OnMessageCompose event not work with Drafts

### ISSUE THREAD

**Issue by frnode** [OfficeDev/office-js#3721]
# Provide required information needed to triage your issue

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Office on the web
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Targeted / preview
* Operating System: Windows 11
* Browser (if using Office on the web): Edge, Chrome, Firefox
 
## Expected behavior

the "OnMessageCompose" event should fire with drafts.

## Current behavior

The event does not fire with drafts. (only on the Web, with PC it works, also with OneOutlook). It was in working order before on the web, we have an application that has not received any updates and now it no longer works.

I've also seen that the launchevent is no longer loaded.

## Steps to reproduce

1. Make "OnMessageCompose" event.
2. Launch Outlook Web
3. Open a Drafts (compose)

## Context

Just a notification.

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by frnode** [comment]
I have some problem with `OnAppointmentOrganizer` , `OnMessageFromChanged` (work on PC, not on web).

**Comment by frnode** [comment]
Hello, do you have any feedback regarding this case? @exextoc I have the impression that this also happens with the standard branch too.

**Comment by exextoc** [comment]
@frnode do other LauchEvents work fine for you?

**Comment by frnode** [comment]
> @frnode do other LauchEvents work fine for you?

This only happens on the OWA. The other events work and even `OnMessageCompose` (during a new message) but during a draft, it does not work, although it should.

**Comment by exextoc** [comment]
Thanks for reporting. We have put this issue in out backlog but unfortunately we don't have any timelines to share at this point.
Internal tracking ID- 235856

**Comment by frnode** [comment]
> Thanks for reporting. We have put this issue in out backlog but unfortunately we don't have any timelines to share at this point. Internal tracking ID- 235856

I'm here to update:

We identified the issue with `OnMessageCompose` and `OnMessageFromChanged`.

Sorry for the inconvenience, but this won't return any errors and would work fine on the Outlook Windows application... (we only had one function that wasn't used, it didn't contain any errors, it just wasn't called, because we didn't need it at the moment, just removing/using it fixed the problem...).

On the other hand we tried other events, we noted that `OnAppointmentFromChanged` and `OnMessageRecipientsChanged` are not triggered on the OWA. On the New Outlook it works.

The other events reported previously work with our code fix.
